jetbrains-core/tst/software/aws/toolkits/jetbrains/core/TestUtils.kt

```diff
@@ -3,35 +3,39 @@
 
 package software.aws.toolkits.jetbrains.core
 
+import com.intellij.openapi.project.Project
 import com.nhaarman.mockitokotlin2.mock
 import software.amazon.awssdk.services.s3.model.Bucket
 import software.aws.toolkits.core.region.AwsRegion
 import software.aws.toolkits.jetbrains.services.ecs.resources.EcsResources
 import software.aws.toolkits.jetbrains.services.s3.resources.S3Resources
 
-fun fillResourceCache(resourceCache: MockResourceCache) {
-    resourceCache.addEntry(
+fun MockResourceCacheRule.fillResourceCache(project: Project) {
+    this.addEntry(
```

Comment:
I'm surprised by the explicit `this` in all of these extension functions - is there a style reason for that? is it something we should adopt more broadly? I typically do not include the self-reference. Do you think doing so would convey better intent?

Reply:
We did it once, and then copied pasted this pattern everywhere...

I do like the intent it reveals but I dont feel strongly either way.

Reply:
np